### PR TITLE
Add ability to  generate DID from seed.

### PR DESCRIPTION
**Pull request by JSAssassin**
addresses #74

### CODE REVIEW

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

File: lib/VeresOneDriver.js [hunk 1]
```diff
@@ -223,15 +223,16 @@ class VeresOneDriver {
    * @param {LDKeyPair} [options.delegateKey] - Capability delegation key pair.
    * @param {LDKeyPair} [options.assertionKey] - Assertion method key pair.
    * @param {LDKeyPair} [options.keyAgreementKey] - Key agreement key pair.
-   *
+   * @param {Uint8Array} [options.seed] - A 32-byte array seed for a
+   *   deterministic key.
    * @returns {Promise<{didDocument: object, keyPairs: Map,
```
Comment (dlongley) [inline lib/VeresOneDriver.js:228]:
```suggestion
   * @param {Uint8Array} [options.seed] - A 32-byte array seed for a
   *   deterministic key.
   *
   * @returns {Promise<{didDocument: object, keyPairs: Map,
```

**Review by dlongley — APPROVED** [review]
Seems ok. Would be good to get another reviewer here to make sure I'm not missing something.

File: test/VeresOneDriver.spec.js [hunk 2]
```diff
@@ -173,6 +176,23 @@ describe('methods/veres-one', () => {
       expect(didDocument.id).to.match(/^did:v1:nym:z.*/);
     });
 
+    it('should generate a DID document from seed', async () => {
+      const seedBytes = (new TextEncoder()).encode(TEST_SEED).slice(0, 32);
```
Comment (aljones15) [inline test/VeresOneDriver.spec.js:180]: This is a really interesting way of generating a 32-byte seed. @davidlehn Do we have a recommended way of generating a 32-byte seed from a string? @JSAssassin I don't think this is wrong, but it just got me wondering how we should be turning strings into 32-byte seeds. Also of note: this code comes from the `did-method-key-js` test suite here: https://github.com/digitalbazaar/did-method-key/blob/602b82f8f3c71d687035e5642f3e32fa2651e2d0/test/driver.spec.js#L201-L213

**Review by aljones15 — APPROVED** [review]
Approving this, but have not actually tested this yet! I have no issues with this being merged into v14.

**Pull request merged**